bc_obps/reporting/tests/service/test_report_verification_service.py

```diff
@@ -1,10 +1,21 @@
+from decimal import Decimal
 from django.test import TestCase
 from model_bakery.baker import make_recipe
 from reporting.service.report_verification_service import ReportVerificationService
 from reporting.schema.report_verification import ReportVerificationIn
+from unittest.mock import patch
+
+from registration.models import Operation
 
 
 class TestReportVerificationService(TestCase):
+    REGULATED_OPERATION_PURPOSES = {
+        Operation.Purposes.OBPS_REGULATED_OPERATION,
+        Operation.Purposes.OPTED_IN_OPERATION,
+        Operation.Purposes.NEW_ENTRANT_OPERATION,
+    }
+    ATTRIBUTABLE_EMISSION_THRESHOLD = Decimal('25000000')
+
```

Comment:
These should probably live in a utils or config place, it's important that the tests use the same source as the service code for these values